### DISCUSSION TITLE
customize  port in Management API URL

### DISCUSSION THREAD

**Discussion by pedinil** [OutlineFoundation/outline-sdk#243] [General]
The Outline manager includes URL management and port configuration.

Based on my interpretation of the code snippet below: "1024 <= num && num < 65536", it appears that the management port ranges from 1024 to 65536 and is assigned randomly.

Is it possible to customize the port and specify a specific port rather than a random one?

The rationale behind this request is that when using Outline as a VPN, some filtering systems may block server IPs if the port used is non-standard, leading to unreachable IPs. However, if we can utilize common ports like 443 or 80, or other well-known portal ports, they are less likely to be filtered.

**Comment by cornzzy** [comment]
It's possible manually: https://github.com/Jigsaw-Code/outline-server/tree/master/src/shadowbox
Then give the token to outline manager. It looks like your target users are in Iran. It gets blocked in Iran under a day specially when حاکم is scared. The [developers deny that](https://www.reddit.com/r/outlinevpn/comments/1cb22sl/comment/l1722td/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) for some reason I don't understand.

https://github.com/Jigsaw-Code/outline-server/issues/1518
https://www.reddit.com/r/outlinevpn/wiki/index/
